common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayImageSource.scala

```diff
@@ -0,0 +1,28 @@
+package uk.ac.wellcome.display.models
+
+import io.circe.generic.extras.JsonKey
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.ac.wellcome.models.work.internal.{Identified, ImageSource}
+
+@Schema(
+  name = "ImageSource",
+  description = "A description of the entity from which an image was sourced"
+)
+case class DisplayImageSource(
+  @Schema(
+    description = "Identifer of the image source"
+  ) id: String,
+  @JsonKey("type") @Schema(
+    name = "type",
+    description = "What kind of source this is"
```

Comment:
Implying multiple kinds are possible? Although this is a very odd field to describe.

Reply:
Yes, there could be multiple sources.... got it.